libs/api/observability/src/lib/trace-wrappers/interceptor-trace-wrapper.ts

```diff
@@ -0,0 +1,27 @@
+import { ModulesContainer } from '@nestjs/core';
+
+import { TraceWrapper } from './abstract-trace-wrapper';
+
+// Wraps all NestJS Interceptors `intercept` methods with OpenTelemetry spans
```

Comment:
```suggestion
/**
 * Wraps all NestJS Interceptors `intercept` methods with OpenTelemetry spans
 */
```

Shows the comment in docs as well.